packages/components/src/date-time/README.md

```diff
@@ -84,3 +84,10 @@ This function will be called on each day, every time user browses into a differe
 
 - Type: `Function`
 - Required: No
+
+### keepOpen
```

Comment:
Thanks for the fix, I have one question here.

Do we really want this prop? I mean it seems sensible to always keep the focus in the date time picker regardless of whether it's inside a popover or not.

Reply:
yeah, that's something we talked about a little bit [here](https://github.com/WordPress/gutenberg/pull/29738#pullrequestreview-609745144). Maybe we should remove the prop.

Reply:
Anyone for a follow-up here ;) ?

Reply:
I think @allilevine will be happy to take it. Otherwise, I can take over too.

Reply:
Yep I can work on a follow-up to remove the prop.

Reply:
Follow-up here: https://github.com/WordPress/gutenberg/pull/29893